src/tests/EdFi.DataManagementService.Tests.E2E/Features/DeleteReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,93 @@
+Feature: Validation of DELETE requests that would cause a foreign key violation
+
+        Background:
+            Given this School Year Type exists
+                  | id                               | schoolYear | currentSchoolYear | schoolYearDescription |
+                  | 20906a639ed04dfa9db77107f2c7834a | 2022       | false             | 2021-2022             |
```

Comment:
There is no way for us to know the `id` value in advance, so that column needs to be deleted in all of the `GIVEN` statements.

Reply:
@stephenfuqua  I'm seeing this in other scenarios(Given existing records with ID XXXX) is that something we need to remove as well?... not now but in other ticket.

Reply:
ID Column removed